runtime/runtime-params-estimator/estimator-warehouse/src/check.rs

```diff
@@ -131,6 +145,41 @@ fn estimation_changes(
     Ok(warnings)
 }
 
+fn estimation_uncertain_changes(
+    db: &Db,
+    estimation_names: &[String],
+    commit_before: &str,
+    commit_after: &str,
+    metric: Metric,
+) -> anyhow::Result<Vec<Notice>> {
+    let mut warnings = Vec::new();
+    for name in estimation_names {
+        let b = &EstimationRow::get(db, name, commit_before, metric)?[0];
+        let a = &EstimationRow::get(db, name, commit_after, metric)?[0];
+        match (&b.uncertain_reason, &a.uncertain_reason) {
+            (None, None) => continue,
+            (Some(uncertain_before), None) => add_warning(&mut warnings, name.clone(), uncertain_before.clone(), "None".to_owned()),
+            (None, Some(uncertain_after)) => add_warning(&mut warnings, name.clone(),"None".to_owned(), uncertain_after.clone()),
+            (Some(uncertain_before), Some(uncertain_after)) => {
+                if !uncertain_before.eq(uncertain_after) {
+                    add_warning(&mut warnings, name.clone(), uncertain_before.clone(), uncertain_after.clone());
+                }
+            },
+        }
+    }
+
+    Ok(warnings)
+}
+
+fn add_warning(warnings: &mut Vec<Notice>, name: String, before: String, after: String) {
+    warnings.push(Notice::UncertainChange(UncertainChange {
+        estimation: name,
+        before: before,
+        after: after,
```

Comment:
```suggestion
        before,
        after,
```
we generally use this syntax in our codebase whenever it is "naturally possible" like in this case

Reply:
fixed